draft-bbs-signatures.md

```diff
@@ -748,6 +753,81 @@ Procedure:
 5. return (scalar_1, ..., scalar_n)
 ```
 
+### OctetsToSignature
+
+This operation describes how to decode an octet string, validate it and return the underlying components that make up the signature.
+
+```
+(A, e, s) = octets_to_signature(signature_octets)
+
+Inputs:
+
+- signature_octets (REQUIRED), octet string of the form output from signature_to_octets operation.
+
+Outputs:
+
+- A, a valid point in the G1 subgroup which is not equal to the identity point.
+- e, a non-negative integer representing a valid scalar value with the range of 0 < e < q.
```

Comment:
Seems a bit redundant?

Reply:
Not sure I understand what you mean here, the description is redundant?

Reply:
IMO although the non-negative part is not really needed i would like to keep it to highlight the fact that e can be un signed integer. But removing it and leaving it as 

```
- e, an integer representing a valid scalar value with the range of 0 < e < q.
```
Is also correct.